src/pages/ContestDetails/ContestDetails.tsx

```diff
@@ -268,6 +271,11 @@ export const ContestDetails = () => {
                   <Text variant="alternate" strong>{`Time left: ${timeLeftString(timeLeft)}`}</Text>
                 </Row>
               )}
+              {["JUDGING", "SHERLOCK_JUDGING", "ESCALATING"].includes(contest.status) && (
```

Comment:
Should we make the "View findings" button visible when the contest is FINISHED as well? The report should be available at that stage, but still having access to the judging repo might be helpful in my opinion

cc @Evert0x

Reply:
Sg